packages/eas-cli/src/metadata/apple/tasks/app-review-detail.ts

```diff
@@ -0,0 +1,57 @@
+import { AppStoreReviewDetail } from '@expo/apple-utils';
+import assert from 'assert';
+import chalk from 'chalk';
+
+import Log from '../../../log';
+import { logAsync } from '../../utils/log';
+import { AppleTask, TaskDownloadOptions, TaskPrepareOptions, TaskUploadOptions } from '../task';
+
+export type AppReviewData = {
+  /** The current app info that should be edited */
+  reviewDetail: AppStoreReviewDetail;
+};
+
+export class AppReviewDetailTask extends AppleTask {
```

Comment:
Maybe add a high-level comment here about what the class does

Reply:
Done